src/amo/components/Addon/index.js

```diff
@@ -452,8 +453,14 @@ export class AddonBase extends React.Component {
       isRestartRequired = addon.isRestartRequired;
     }
 
+    const addonsByAuthorsLength = addonsByAuthors ? addonsByAuthors.length : 0;
+    const _classNames = classNames('Addon', `Addon-${addonType}`, {
+      'Addon--has-more-addons': addonsByAuthorsLength > 0,
+      'Addon--has-many-more-addons': addonsByAuthorsLength > 3,
+    });
+
     return (
-      <div className={classNames('Addon', `Addon-${addonType}`)}>
+      <div className={_classNames}>
```

Comment:
I don't like the `_` vars as I think of them as private variables. Seems we could be doing naming better here (`cssClasses` would be fine) but in this case why bother? We do multi-line, inline `classNames()` calls [elsewhere in the code](https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/CategoryHeader/index.js#L50) and I'm not the biggest fan of assigning a variable that's only used once.

I think this could just be multi-line like it is elsewhere.